simpleflow/executor.py

```diff
@@ -116,8 +122,8 @@ def before_replay(self):
     def after_replay(self):
         return self._workflow.after_replay(self._history)
 
-    def after_finished(self):
-        return self._workflow.after_finished(self._history)
+    def after_closed(self):
+        return self._workflow.after_closed(self._history)
```

Comment:
There are other occurrences of `after_finished()` left, they should be renamed too.